dbt-cta/stripe/models/1_cta_full_refresh/balance_transactions_fee_details_base.sql

```diff
@@ -1,3 +1,13 @@
+{% set partitions_to_replace = [
```

Comment:
looks like this `set partitions_to_replace` block of code is in here twice for all the base models

Reply:
Oh!! yep I know exactly why that happened... fixing now

Reply:
Seems like this is fixed now. If it passes dbt tests locally, this has my stamp of approval.